src/video.js

```diff
@@ -64,3 +69,59 @@ export const checkVideoBidSetup = hook('sync', function(bid, adUnit, videoMediaT
 
   return true;
 }, 'checkVideoBidSetup');
+
+export function registerVastTrackers(moduleType, moduleName, trackerFn) {
+  if (typeof trackerFn == 'function') { vastTrackers.push({'moduleType': moduleType, 'moduleName': moduleName, 'trackerFn': trackerFn}); }
+}
+
+export function insertVastTrackers(trackers, vastXml) {
+  const doc = new DOMParser().parseFromString(vastXml, 'text/xml');
```

Comment:
no need to instantiate a new DOMParser at every insertion. I wonder if it has a performance cost, might be best to keep in memory.

Reply:
@dgirardi any thoughts on this ?

Reply:
I don't know either. My instinct is that this version is better though:

 - if DOMParser has internal state, we don't want to reuse it;
 - if it doesn't, the constructor is useless, and I'll bet you a beer that the VM does not actually instantiate anything in reality - they tend to be pretty well optimized.

Reply:
On my side, I have no idea, I'll do the way you want. I guess in order to reuse it I would do something like :
`export function insertVastTrackers(trackers, vastXml, dom, xml)`
I don't know if it would be very clean to get them forwarded this way, unless you had something else in mind ?

Reply:
Could you please confirm me if that's okay for you to make the way I thought about or should I make some other way or leave it as it is ? Thanks